http2/src/solicit/mod.rs

```diff
@@ -404,6 +412,7 @@ impl Error for HttpError {
             HttpError::UnknownStreamId => "Attempted an operation with an unknown HTTP/2 stream ID",
             HttpError::UnableToConnect => "An error attempting to establish an HTTP/2 connection",
             HttpError::MalformedResponse => "The received response was malformed",
+            HttpError::Timeout => "Connection time out",
```

Comment:
Either enum value should be called `ConnectTimeout` (or smth like that), or message should say just "Timeout".